AndelaOSP/art-backend
# show categorization on asset logs response

[Finishes #166728438]

#### What does this PR do?
show categorization on asset logs response


#### Description of Task to be completed?
an asset log will show details about the asset it belongs to like asset category, asset make, asset type and asset subcategory type


#### How should this be manually tested?
n postman, go to http://127.0.0.1:8000/api/v1/asset-logs You will see more asset details of that specific asset log.



#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2146417/stories/166728438
